app/src/main/java/fr/free/nrw/commons/theme/NavigationBaseActivity.java

```diff
@@ -135,9 +135,23 @@ private void setUserName() {
             username.setText(allAccounts[0].name);
         }
         ImageView userIcon = navHeaderView.findViewById(R.id.user_icon);
-        userIcon.setOnClickListener(v -> {
+        makeClickable(userIcon, NavigationBaseActivity.this, AchievementsActivity.class);
```

Comment:
Instead of adding event listeners on both the views separately wrap the icon and name in a Linearlayout and add a listener to it.